_sass/brand.scss

```diff
@@ -1,475 +1,328 @@
 html,
 body {
-  font-family: "Qanelas Soft", "Open Sans", sans-serif;
-  box-sizing: border-box;
-  max-width: 100%;
+    font-family: "Open Sans", sans-serif;
```

Comment:
Good eye, regarding the swap to Open Sans for the body.

@Rexford74 would be proud.